state/application.go

```diff
@@ -84,9 +84,11 @@ type applicationDoc struct {
 	// be relatively straight forward, but very time consuming.
 	// When moving to CharmHub or removing CharmStore from Juju it should be
 	// tackled then.
-	CharmURL             *string      `bson:"charmurl"`
-	Channel              string       `bson:"cs-channel"`
-	CharmOrigin          *CharmOrigin `bson:"charm-origin"`
+	CharmURL    *string      `bson:"charmurl"`
+	Channel     string       `bson:"cs-channel"`
```

Comment:
sad panda that we can't easily get rid of "cs"

Reply:
It's gone from here in 3.1, this is 2.9 where it's still needed. This is a spacing change with the comment.